Include/object.h

```diff
@@ -675,8 +672,14 @@ given type object has a specified feature.
 /* NOTE: The following flags reuse lower bits (removed as part of the
  * Python 3.0 transition). */
 
+/* The following flags are kept for compatibility, but all types are
+ * supposed to support the given functionality.  Starting with 3.7,
```

Comment:
This explicitly mentions the CPython release of the change and needs adaptation when this gets merged.

Reply:
I think it should also be mentioned that Py_TPFLAGS_HAVE_FINALIZE is being rolled into the always-supported default, if that effects how to write objects.  If it does not affect anything extension writers do, then that (and the reason it doesn't) should be mentioned.

Reply:
Conversely, if it does affect anything for extension writers -- even just simplifying things -- that should be added to the porting section of what's new.